### DISCUSSION TITLE
Wireless energy measurement module unable to use data in HA energy dashboard

### DISCUSSION THREAD

**Discussion by jefG-ha** [joleys/niko-home-control-II#187] [Q&A] [answered]
the data from the wireless measurement module is put into HA in W or kW but not in Wh or kWh as required by the home assistant energy dashboard.

Configure grid consumption:
Grid consumption is the energy that flows from the energy grid to your home.
Pick a sensor which measures grid consumption in either of **cal, Gcal, GJ, J, kcal, kJ, kWh, Mcal, MJ, MWh, Wh.**

**Comment by jefG-ha** [comment]
![image](https://github.com/user-attachments/assets/e9451eda-1613-4560-b8ea-b9040013165b)
![image](https://github.com/user-attachments/assets/d4a2d610-76fc-40c8-a16b-147362416972)
![image](https://github.com/user-attachments/assets/64ba45ee-1c36-44db-ae4d-89fc3d9b452b)

some screenshot to show the issue

**Comment by joleys** [comment, marked as answer]
Please check the [ReadMe](https://github.com/joleys/niko-home-control-II?tab=readme-ov-file#calculated-totals).  This will guide you through the required configuration.

**Reply by jeroenmaes** [reply]
@joleys would it be interesting to add this out of the box for every power entity? Integrations like Shelly have this out of the box.
